spec/features/create_anschutz_work_spec.rb

```diff
@@ -1,20 +1,102 @@
 # frozen_string_literal: true
 require 'rails_helper'
-require File.expand_path('../helpers/create_anschutz_work_user_context', __dir__)
-
 include Warden::Test::Helpers
 
 # NOTE: If you generated more than one work, you have to set "js: true"
 RSpec.feature 'Create a AnschutzWork', js: false do
-  include_context 'create anschutz work user context' do
-    let(:work_type) { "anschutz_work" }
-
-    scenario do
-      visit_new_work_page
-      add_files_to_work
-      add_metadata_to_work
-      apply_work_visibility
-      check_agreement_and_submit
+  let(:user) { User.new(email: 'test@example.com') { |u| u.save(validate: false) } }
+  let(:admin_set_id) { AdminSet.find_or_create_default_admin_set_id }
+  let(:permission_template) { Hyrax::PermissionTemplate.find_or_create_by!(source_id: admin_set_id) }
+  let(:workflow) { Sipity::Workflow.create!(active: true, name: 'test-workflow', permission_template: permission_template) }
+  let(:work_type) { "anschutz_work" }
+
+  before do
+    # Create a single action that can be taken
+    Sipity::WorkflowAction.create!(name: 'submit', workflow: workflow)
+
+    # Grant the user access to deposit into the admin set.
+    Hyrax::PermissionTemplateAccess.create!(
+      permission_template_id: permission_template.id,
+      agent_type: 'user',
+      agent_id: user.user_key,
+      access: 'deposit'
+    )
+    login_as user
+  end
+
+  it 'renders the new Anschutz Work page' do
+    visit "concern/#{work_type.to_s.pluralize}/new"
+    expect(page).to have_content "Add New Anschutz Work"
+  end
+
+  it 'adds files to work' do
+    visit "concern/#{work_type.to_s.pluralize}/new"
+    click_link "Files" # switch tab
+    expect(page).to have_content "Add files"
+    expect(page).to have_content "Add folder"
+    within('span#addfiles') do
+      attach_file("files[]", Rails.root.join('spec', 'fixtures', 'hyrax', 'image.jp2'), visible: false)
+      attach_file("files[]", Rails.root.join('spec', 'fixtures', 'hyrax', 'jp2_fits.xml'), visible: false)
+    end
+  end
+
+  it 'applys work visibility' do
+    visit "concern/#{work_type.to_s.pluralize}/new"
+    visibility = :open
+    find('body').click
+    choose("#{work_type}_visibility_#{visibility}")
+    case visibility
+    when :open
+      expect(page).to have_content('Please note, making something visible to the world (i.e. marking this as Public) may be viewed as publishing which could impact your ability to')
+    end
+  end
+
+  it 'saves the work' do
+    visit "concern/#{work_type.to_s.pluralize}/new"
```

Comment:
We are using the same string URL for all of the specs, so instead, lets make it a `let(:new_work_path)` and then we can just have `visit new_work_path` and make it easier to parse.